webui/react/src/components/LogViewer.module.scss

```diff
@@ -117,6 +117,7 @@
       width: 100%;
     }
   }
+  .scrollToTop.enabled,
```

Comment:
**nit:** swap the order to be alphabetical
```
.enableTailing.enabled,
.scrollToTop.enabled {
```

Reply:
Fixed (push soon). Btw I was wondering if you prefer to have those 2 components sharing the same SCSS module (file) or if it's better to create a copy (to have it decoupled).

Reply:
I think referencing the original one is good, just because the plan is to go back to being one log viewer eventually. It's only split just because of the endpoint responses are not unified yet between the different log types.